cluster/store/store.go

```diff
@@ -276,11 +277,11 @@ func (st *Store) onLeaderFound(timeout time.Duration) {
 				return fmt.Errorf("load schema: %w", err)
 			}
 
-			// If the legacy schema is empty we can abort early
-			if len(legacySchema) == 0 {
-				st.log.Info("legacy schema is empty, nothing to migrate")
-				return nil
-			}
+			// // If the legacy schema is empty we can abort early
```

Comment:
We should keep this condition here to avoid doing extra work in case we have nothing to migrate.

Reply:
Done